src/QualificationRunner/Program.cs

```diff
@@ -36,40 +35,37 @@ static int Main(string[] args)
 
       private static void startCommand<TRunOptions>(CLICommand<TRunOptions> command)
       {
-         var (logger, loggerFactory) = initializeLogger(command);
+         var logger = initializeLogger(command);
 
          logger.AddInfo($"Starting {command.Name.ToLower()}");
          logger.AddDebug($"Arguments:\n{command}");
 
          var runner = IoC.Resolve<IBatchRunner<TRunOptions>>();
 
-         using (loggerFactory)
+         try
          {
-            try
-            {
-               runner.RunBatchAsync(command.ToRunOptions()).Wait();
-               logger.AddInfo($"{command.Name} finished");
-            }
-            catch (Exception e)
-            {
-               logger.AddException(e);
-               logger.AddError($"{command.Name} failed");
-               _valid = false;
-            }
+            runner.RunBatchAsync(command.ToRunOptions()).Wait();
+            logger.AddInfo($"{command.Name} finished");
+         }
+         catch (Exception e)
+         {
+            logger.AddException(e);
+            logger.AddError($"{command.Name} failed");
+            _valid = false;
          }
       }
 
-      private static (ILogger, ILoggerFactory) initializeLogger(CLICommand runCommand)
+      private static IOSPSuiteLogger initializeLogger(CLICommand runCommand)
       {
-         var loggerFactory = IoC.Resolve<ILoggerFactory>();
+         var loggerCreator = IoC.Resolve<ILoggerCreator>();
+
 
-         loggerFactory
-            .AddConsole(runCommand.LogLevel);
+         loggerCreator.AddLoggingBuilderConfiguration(x => x.SetMinimumLevel(runCommand.LogLevel).AddConsole());
 
          if (!string.IsNullOrEmpty(runCommand.LogFileFullPath))
-            loggerFactory.AddFile(runCommand.LogFileFullPath, runCommand.LogLevel, runCommand.AppendToLog);

```

Comment:
@abdelr @georgeDaskalakis can you confirm that this is the right way to update? 
What happened with our `appendToLog` parameter?

Reply:
@msevestre With serilog as a provider there is no more the possibility to overwrite the file - the only option is to append. Of course this could easily be handled, since AddFile is an extension method from us: we could add a boolean to its signature and according to its value clear the file before creating the logger. 

However serilog does provide the possibility to roll over the log. A FileSizeLimit can be specified and on reaching this limit the existing log is archived. Would it make sense to use this feature instead?

Reply:
I think the use case was not about FileSizeLimit. 
It is to make sure that the log file is cleared when starting a new QualiRunner to avoid outdated stuff. This is probably not a b ig deal but this is weird that we decided to simply remove the option